client/my-sites/themes/single-site.jsx

```diff
@@ -184,7 +184,7 @@ export default connect(
 		{
 			queryParams: ThemesListSelectors.getQueryParams( state ),
 			themesList: ThemesListSelectors.getThemesList( state ),
-			selectedSite: getSelectedSite( state )
+			selectedSite: getSelectedSite( state ) || false,
```

Comment:
> Seems like something we should consider fixing one way or the other.

It was a [conscious decision](https://github.com/Automattic/wp-calypso/pull/2657#discussion_r50544162) to move to `null`. I feel that the showcase code has become hardened over time to work with `false`, so this is a pragmatic compromise.